src/browser-provider/browser.js

```diff
@@ -0,0 +1,60 @@
+import viewportSizes from 'viewportsizes';
```

Comment:
I'd rather go with our own list implementation from `browser natives`, because this one fetches data on install, which may break testcafe installation and require additional install-time only dependencies.